<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotGui::TaskOTAttacher</class>
 <widget class="QWidget" name="RobotGui::TaskOTAttacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="message">
     <property name="text">
      <string>Selection accepted</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="buttonParentFrame">
       <property name="text">
        <string>ParentFrame</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineParentFrame"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Anchor activation status:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxAttachmentActivate">
      <property name="text">
       <string>activated</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_AttachmentOffset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Anchor Offset (in local coordinates):</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelOffset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>In x-direction:</string>
        </property>
        <property name="buddy">
         <cstring>labelOffset</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelOffset2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>In y-direction:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="Gui::PrefQuantitySpinBox" name="attachmentOffsetY" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>5</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Note: The placement is expressed in local coordinate system
of object being attached.</string>
        </property>
        <property name="keyboardTracking" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelOffset3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>In z-direction:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Gui::PrefQuantitySpinBox" name="attachmentOffsetZ" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>5</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Note: The placement is expressed in local coordinate system
of object being attached.</string>
        </property>
        <property name="keyboardTracking" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelYaw">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Around x-axis:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelPitch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Around y-axis:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelRoll">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Around z-axis:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Gui::PrefQuantitySpinBox" name="attachmentOffsetX" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>5</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Note: The placement is expressed in local coordinate system
of object being attached.</string>
        </property>
        <property name="keyboardTracking" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="Gui::QuantitySpinBox" name="attachmentOffsetRoll" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Rotation around the x-axis
Note: The placement is expressed in local coordinate system
of object being attached.</string>
        </property>
        <property name="keyboardTracking" stdset="0">
         <bool>false</bool>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">deg</string>
        </property>
        <property name="minimum" stdset="0">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="Gui::QuantitySpinBox" name="attachmentOffsetPitch" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Rotation around the y-axis
Note: The placement is expressed in local coordinate system
of object being attached.</string>
        </property>
        <property name="keyboardTracking" stdset="0">
         <bool>false</bool>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">deg</string>
        </property>
        <property name="minimum" stdset="0">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="Gui::QuantitySpinBox" name="attachmentOffsetYaw" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Rotation around the z-axis
Note: The placement is expressed in local coordinate system
of object being attached.</string>
        </property>
        <property name="keyboardTracking" stdset="0">
         <bool>false</bool>
        </property>
        <property name="unit" stdset="0">
         <string notr="true">deg</string>
        </property>
        <property name="minimum" stdset="0">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum" stdset="0">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxUseCurrentTransform">
     <property name="toolTip">
      <string>Flip side of attachment and offset</string>
     </property>
     <property name="text">
      <string>Use current transform as anchor offset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefQuantitySpinBox</class>
   <extends>Gui::QuantitySpinBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonParentFrame</tabstop>
  <tabstop>lineParentFrame</tabstop>
  <tabstop>attachmentOffsetX</tabstop>
  <tabstop>attachmentOffsetY</tabstop>
  <tabstop>attachmentOffsetZ</tabstop>
  <tabstop>attachmentOffsetRoll</tabstop>
  <tabstop>attachmentOffsetPitch</tabstop>
  <tabstop>attachmentOffsetYaw</tabstop>
  <tabstop>checkBoxUseCurrentTransform</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
